app/views/parks/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="index">
+<h1>United States National Parks</h1>
+
+<button><%= link_to('Add a Park', new_park_path)%></button>
+<% @parks.each do |park| %>
+  <h4><%= link_to(park.name, park_path(park))%></h4>
+  <p><%= park.description %></p>
+  <p><%= park.picture %></p>
```

Comment:
A better way to deal with the image link would be to add it to an image tag
<img src='<%= park.picture %>' >
so you see it rather than the link